src/pyhub/llm/agents/tools/code_interpreter/session.py

```
@@ -0,0 +1,229 @@
+"""Session management for code execution."""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)